webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js

```diff
@@ -55,7 +56,7 @@ const actions = ({
   },
   abort: {
     name: 'template-invocation-abort-job',
-    text: __('Abort task'),
+    text: __('Abort Task'),
```

Comment:
Late to the party, but I don't think we use Title Case anymore. The reason I now see it is because it changed a string in the translations. Should we revert this specific line?

Reply:
Do we state somewhere what casing should be used?

Reply:
https://www.patternfly.org/ux-writing/capitalization

Reply:
https://github.com/theforeman/foreman_remote_execution/pull/959